r4e-image.sh

```diff
@@ -86,7 +86,7 @@ export HTTP_API
 
 # prepare local variables
 IMAGE_BASE_URL=http://$IMAGE_SERVER_ADDRESS/$IMAGE_NAME
-IMAGE_FOLDER=/var/www/html/$IMAGE_NAME
+IMAGE_FOLDER=/var/www/html/${IMAGE_NAME}_$(date +'%m_%d_%Y-%H_%M')
```

Comment:
my concern with this one that it will be less expected than what the user specified. I'd prefer cleaning the folder as part of the clean up. WDYT?

Reply:
I also thought like that at first
but then after several runs the image was removed but the script fails and it can be very frustrating

Reply:
If the user wish to preserve the image, the user can specify a different image-name for the new attempt.